frontend/src/components/layout/Menu/MenuContainer.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import {
+  AccountCircleOutlined,
+  ArticleOutlined,
+  FolderSpecialOutlined,
+  AddToQueueOutlined,
+  EmojiPeopleOutlined,
+  AssessmentOutlined,
+  QuizOutlined,
+} from '@mui/icons-material';
+
+import PATHS from 'constants/routes';
+
+import Menu from './Menu';
+
+const EMPLOYEE_MENU = [
+  {
+    path: PATHS.profile,
+    title: 'Profile',
+    icon: <AccountCircleOutlined fontSize="large" />,
+  },
+  {
+    path: PATHS.coursesList,
+    title: 'Courses List',
+    icon: <ArticleOutlined fontSize="large" />,
+  },
+  {
+    path: PATHS.myCourses,
+    title: 'My Courses',
+    icon: <AddToQueueOutlined fontSize="large" />,
+  },
+  {
+    path: PATHS.help,
+    title: 'Help',
+    icon: <QuizOutlined fontSize="large" />,
+  },
+];
+
+const MANAGER_MENU = [
+  {
+    path: PATHS.profile,
+    title: 'Profile',
+    icon: <AccountCircleOutlined fontSize="large" />,
+  },
+  {
+    path: PATHS.coursesList,
+    title: 'Courses List',
+    icon: <ArticleOutlined fontSize="large" />,
+  },
+  {
+    path: PATHS.myCourses,
+    title: 'My Courses',
+    icon: <FolderSpecialOutlined fontSize="large" />,
+  },
+  {
+    path: PATHS.requests,
+    title: 'Pending Requests',
+    icon: <AddToQueueOutlined fontSize="large" />,
+  },
+  {
+    path: PATHS.employees,
+    title: 'Employees',
+    icon: <EmojiPeopleOutlined fontSize="large" />,
+  },
+  {
+    path: PATHS.help,
+    title: 'Help',
+    icon: <QuizOutlined fontSize="large" />,
+  },
+];
+
+const ADMIN_MENU = [
+  {
+    path: PATHS.coursesList,
+    title: 'Courses List',
+    icon: <ArticleOutlined fontSize="large" />,
+  },
+  {
+    path: PATHS.skills,
+    title: 'Skills',
+    icon: <AssessmentOutlined fontSize="large" />,
+  },
+  {
+    path: PATHS.help,
+    title: 'Help',
+    icon: <QuizOutlined fontSize="large" />,
+  },
+];
+
+const ROLES_MENU = {
+  employee: EMPLOYEE_MENU,
+  manager: MANAGER_MENU,
+  admin: ADMIN_MENU,
+};
+
+export const menuItems = ROLES_MENU['employee'];
```

Comment:
Remove please export. We can't export anything except container inside container file. And also we shouldn't import inside component anything from its container(imports hell)
Move please all the uppercase constants to the constants folder.
Let's define Container like:
```
const MenuContainer: React.FC = () => {
  const menuItems = ROLES_MENU['employee'];

  return <Menu menuList={menuItems}/>;
}
```